### PR TITLE
Meeting notes 2018-05-03

### LINKED ISSUE #300: Node.js Foundation Community Committee Meeting 2018-05-03
## Time

**UTC Thu 03-May-2018 16:00 (04:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Thu 03-May-2018 09:00 (09:00 AM) |
| US / Mountain | Thu 03-May-2018 10:00 (10:00 AM) |
| US / Central  | Thu 03-May-2018 11:00 (11:00 AM) |
| US / Eastern  | Thu 03-May-2018 12:00 (12:00 PM) |
| London        | Thu 03-May-2018 17:00 (05:00 PM) |
| Amsterdam     | Thu 03-May-2018 18:00 (06:00 PM) |
| Moscow        | Thu 03-May-2018 19:00 (07:00 PM) |
| Chennai       | Thu 03-May-2018 21:30 (09:30 PM) |
| Hangzhou      | Fri 04-May-2018 00:00 (12:00 AM) |
| Tokyo         | Fri 04-May-2018 01:00 (01:00 AM) |
| Sydney        | Fri 04-May-2018 02:00 (02:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Community%20Committee+Meeting+2018-05-03&iso=20180503T16
* or http://www.wolframalpha.com/input/?i=04PM+UTC%2C+May+03%2C+2018+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1B1niZ3uIDEtOdksQgIjWH_M0vhcjNmX9PFSuz9NajUA/edit>

## Agenda

Extracted from **cc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/build

* Bots + Apps in the Org [#1247](https://github.com/nodejs/build/issues/1247)

### nodejs/TSC

* Wiki Management [#525](https://github.com/nodejs/TSC/issues/525)

### nodejs/admin

* Travel Fund approval and being careful about funds available [#99](https://github.com/nodejs/admin/issues/99)

### nodejs/community-committee

* CommComm Membership: Call For Feedback on New Definitions [#276](https://github.com/nodejs/community-committee/issues/276)
* Record and serve the meetings in audio format  [#232](https://github.com/nodejs/community-committee/issues/232)

### nodejs/user-feedback

* 2018-04-27 Public User Feedback Meeting - Enterprise Use Case [#52](https://github.com/nodejs/user-feedback/issues/52)

### nodejs/mentorship

* We need more active members of the WG [#32](https://github.com/nodejs/mentorship/issues/32)
* Set a Code of Conduct for mentors and mentees [#31](https://github.com/nodejs/mentorship/issues/31)
* Create a mentorship subgroup focused on to identify 6 month projects in Node.js [#30](https://github.com/nodejs/mentorship/issues/30)

## Invited

* CommComm Members: @nodejs/community-committee

### Observers
* @DiegoZoracKy (Diego ZoracKy - observer)
* @ZibbyKeaton (Zibby Keaton - observer)
* @Bamieh (Ahmad Bamieh - observer)
* @pchrysa (Chrysa - observer)
* @foadlab (observer)
* @codeekage (Abraham Agiri - observer)
* @Voskan (Voskan Voskanyan - observer)
* @waleedashraf (Waleed Ashraf - observer)
* @maddhruv (Dhruv Jain - observer)
* @elainechan (Elaine Chan - observer)
* @Jiasm (贾顺名(shunming jia) - guest)
* @detrohutt (guest)

## Notes

The agenda comes from issues labelled with `cc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* link for participants: Will be added a few minutes before meeting starts
* For those who just want to watch: 
* youtube admin page: https://www.youtube.com/my_live_events?filter=scheduled

### PR DESCRIPTION
**Pull request by JemBijoux**
Closes #300

### CODE REVIEW

File: meetings/2018-05-03.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+- There were no objections, so it was pushed to the board
+- There will still be other todos and PR’s that come out of this. (emeriti needs further detail, errata pr)
+- Probably other work that needs to happen around working groups, and empowering other groups (working groups)
+- When big changes happen from a sub committee, we document it in some sort of history file so we have a record. (HISTORY.md? Changelog?)
+
+### nodejs/mentorship: "Priority Document" outlining areas of greatest need for Node.js ecosystem
+
+- Is there a document about best things to work on for ecosystem
+- created a plan for what they though it should look like
+- https://github.com/nodejs/mentorship/issues/38#issuecomment-386313778
```
Comment (detrohutt) [inline meetings/2018-05-03.md]: Just https://github.com/nodejs/mentorship/issues/38 is fine. The comment linked to is irrelevant.

File: meetings/2018-05-03.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+- [Record and serve the meetings in audio format](https://github.com/nodejs/community-committee/issues/232)
+- [2018-04-27 Public User Feedback Meeting - Enterprise Use Case](https://github.com/nodejs/user-feedback/issues/52)
+- [We need more active members of the WG](https://github.com/nodejs/mentorship/issues/32)
+- [Set a Code of Conduct for mentors and mentees](https://github.com/nodejs/mentorship/issues/31)
+- [Create a mentorship subgroup focused on to identify 6 month projects in Node.js](https://github.com/nodejs/mentorship/issues/30)
+
+
+## Initiatives
+
+### User Feedabck
```
Comment (detrohutt) [inline meetings/2018-05-03.md]: Feedback*

File: meetings/2018-05-03.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+- The wiki was out of date; had stale information
+- Some information on nodejs.org, not great to have two different places for information
+- (may have also been triggered by some wiki vandalism)
+- Wiki may already be disabled and we’ll all converge on one spot: if you know of something useful pull it over
+- Might want to add an issue to the website redesign repo (or a link to some getting started content)
+
+### nodejs/admin: Travel Fund approval and being careful about funds available #99
+
+- Nothing new to report here
+- Might need to define a treasurer on the tsc side, Jem is still volunteering from
```
Comment (detrohutt) [inline meetings/2018-05-03.md]: from...? Incomplete sentence?

File: meetings/2018-05-03.md (hunk 1 above)
Comment (detrohutt) [reply meetings/2018-05-03.md]: Also, should be "what they thought* "

File: meetings/2018-05-03.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+- Wireframes that we currently have wouldn’t work well with the default “linux foundation” template. Unification would be good, but not essential?
+
+## Discussion
+
+### nodejs/build: Bots + Apps in the Org #1247
+
+- Has come up a few times recently (unfortunately not all stakeholders on call)
+- We want to use bots in the github org for one purpose or another (website redesign, i18n, etc)
+- Work was done previously around that, how do we go forward with this?
+- Can come down to “what priviledges does the bot need”? If it’s something we need to do outside of what we’ve normally done then need to explore.
```
[the remark this answers is not in the document]
Comment (JemBijoux) [reply meetings/2018-05-03.md]: Oy, this is one word I *always* spell wrong. Thanks.

**Comment by JemBijoux** [comment]
Thanks for the fixes and feedback as always. Typing during the meeting can be a bit hurried, so a second (or third) set of eyes is always appreciated.

**Pull request merged**